config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "sql_db":{
+
+    },
+    "http_api":{
+        "port": 8000,
+        "jwt_secret":"secret"
```

Comment:
How would you go about configuring the app secrets in production environment?

Reply:
Actually, I forgot to delete the config.json. 
I don't know what I was thinking when I created the config.json because it is a bad practice.

All the configurations are in .env and .env.docker (to be used from docker-compose or from independent docker image).
The reason that I prefer .env for production is that it moves the configurations to the environment of the OS for the executable only, making it very hard for the hacker to obtain them.
So in production, we can delete the secret from .env and add it in another way to the OS environment of the executable which is more secure, for example Kubernetes "Secrets" are similar to ConfigMaps but more secure.